android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -979,57 +1015,23 @@ private void onIsolinesStateChanged(@NonNull IsolinesState type)
   @Override
   protected void onNewIntent(Intent intent)
   {
-    super.onNewIntent(intent);
+    // {@link see BaseMwmFragmentActivity.onCreate()}
+    final Intent initialIntent = intent.getParcelableExtra(SplashActivity.EXTRA_INITIAL_INTENT, Intent.class);
```

Comment:
Android Lint tell this is only available with API 33 and more

Reply:
Fixed by using IntentCompat.getParcelableExtra() in all places across the project.